test.runsettings

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RunSettings>
+  <RunConfiguration>
+    <EnvironmentVariables>
+        <DynamoDb_LocalMode>true</DynamoDb_LocalMode>
+        <DynamoDb_LocalServiceUrl>http://localhost:8000</DynamoDb_LocalServiceUrl>
+        <AWS_REGION>eu-west-2</AWS_REGION>
+        <AWS_ACCESS_KEY_ID>local</AWS_ACCESS_KEY_ID>
+        <AWS_SECRET_ACCESS_KEY>local</AWS_SECRET_ACCESS_KEY>
+    </EnvironmentVariables>
+  </RunConfiguration>
+</RunSettings>
```

Comment:
Debugging these tests outside Docker was quite painful, but this config helped a fair bit to set the correct environment